cloudstate/tests/test_cloudstate.py

```diff
@@ -0,0 +1,7 @@
+# content of test_sample.py
+def inc(x):
+    return x + 1
+
+
+def test_answer():
```

Comment:
I need to resolve this here. If anyone has an idea for a simple test feel free to include